pkg/cloudprovider/providers/gce/gce_clusters.go

```diff
@@ -18,6 +18,9 @@ package gce
 
 import (
 	"context"
+	"errors"
+	"fmt"
+	"github.com/golang/glog"
```

Comment:
technically there should be blank lines separating these imports into three sections: std library, internal deps, external deps

Reply:
Fixed.